includes/filter_meetings.php

```diff
@@ -93,6 +93,7 @@ public function __construct($arguments)
                 $this->attendance_option[] = 'hybrid';
             }
             if (!empty(array_intersect($this->attendance_option, Array('active')))) {
+                $this->attendance_option = [];
```

Comment:
@tech2serve `[]` will fail on some versions of PHP. better to use `array()`

Reply:
Fixed. Thanks.